pkg/types/installconfig.go

```diff
@@ -260,7 +260,9 @@ type ClusterNetworkEntry struct {
 	CIDR ipnet.IPNet `json:"cidr"`
 
 	// HostPrefix is the prefix size to allocate to each node from the CIDR.
-	// For example, 24 would allocate 2^8=256 adresses to each node.
+	// For example, 24 would allocate 2^8=256 adresses to each node. If this
+	// field is not used by the plugin, it can be left unset.
+	// +optional
 	HostPrefix int32 `json:"hostPrefix"`
```

Comment:
`,omitempty`

(otherwise I believe that the copy of install-config.yaml that gets copied to a configmap in the cluster will end up with `hostPrefix: 0` in it)

Reply:
ah dammit, I didn't realise I missed this spot. I thought it was only there in the api repo.